### PR TITLE
fix: upgrade helm chart with scanjob bug fix

### PR DESCRIPTION
**Pull request by niv1612**

### CODE REVIEW

File: charts/datree-admission-webhook/templates/cluster-scan-cronjob.yaml [hunk 1]
```diff
@@ -39,11 +39,28 @@ spec:
...
+    spec:
+      ttlSecondsAfterFinished: {{.Values.scanJob.ttlSecondsAfterFinished | default "100" }}
+      template:      
+        spec:
+          containers:
+          - name: scan-job
+            image: {{.Values.scanJob.image.repository | default "datree/scan-job" }}:{{.Values.scanJob.image.tag | default "0.0.14" }}
+          restartPolicy: {{.Values.scanJob.restartPolicy | default "Never" }}
+  
 ---
```
Comment (TzlilSwimmer123) [inline charts/datree-admission-webhook/templates/cluster-scan-cronjob.yaml:52]: let's put it in the tpl

Comment (myishay) [reply charts/datree-admission-webhook/templates/cluster-scan-cronjob.yaml:52]: +1

File: charts/datree-admission-webhook/values.yaml [hunk 2]
```diff
@@ -61,11 +61,13 @@ affinity: {}
 tolerations: []
 scanJob:
   resources: {}
+  ttlSecondsAfterFinished: 100
   image:
```
Comment (royhadad) [inline charts/datree-admission-webhook/values.yaml:65]: what bug does this fix?

File: charts/datree-admission-webhook/templates/cluster-scan-cronjob.yaml (hunk 1 above)
Comment (royhadad) [reply charts/datree-admission-webhook/templates/cluster-scan-cronjob.yaml:52]: +2

**Review by royhadad — CHANGES_REQUESTED** [review]

File: charts/datree-admission-webhook/values.yaml (hunk 2 above)
Comment (TzlilSwimmer123) [reply charts/datree-admission-webhook/values.yaml:65]: https://codelaborate.atlassian.net/jira/software/c/projects/DAT/boards/10?modal=detail&selectedIssue=DAT-4664

**Comment by niv1612** [comment]
The fix has been released from a different branch

**Pull request closed without merging**